### PR TITLE
Adds make nuke-cluster and create-cluster

### PR DESCRIPTION
**Pull request by professor**
Sometimes you just want a way to quickly start over.

Signed-off-by: Todd Sedano <tsedano@pivotal.io>

### CODE REVIEW

**Comment by hsyuan** [comment]
There is `make clean`, which can delete the demo cluster.

`create-cluster` is too specific. Without `with-python`, `with-perl`, `with-libxml` option, it won't pass ICW. It also assume that ccache is already installed.

I don't recommend push in this changes. Instead, you can create your local customized script.

**Comment by professor** [comment]
We need a reliable way to remove greenplumb from a system. I'm open to
suggestions and alternatives.

I've seen too many times where make clean is not sufficient.

**Comment by danielgustafsson** [comment]
If a make target isn't working we should fix that, what sort of state is it we haven't been able to recover from?

**Comment by jimmyyih** [comment]
-1 from me as well for the points Daniel commented on.  The bash scripts are too specific as if they were for only one developer environment... more directed to the create-cluster script (what if I don't have 8 cores and ccache).

File: gpAux/gpdemo/create-cluster.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -x
+
+pushd ../..
+make clean
+make distclean
+./configure --prefix `pwd`/greenplum-dev --enable-debug CC="ccache gcc"
```
Comment (larham) [inline gpAux/gpdemo/create-cluster.sh]: The standard location for open-source greenplum install is /usr/local/gpdb .  Could this be used?

**Comment by professor** [comment]
The team will try using the delete cluster script again to see what is missing. I'll close this pull request for now.

**Comment by cjcjameson** [comment]
Would it be helpful to people to have `make cluster`, a target in gpAux/gpdemo, available at the top level? The top-level makefile has many targets that simply call out to subdirectories' targets.

**Comment by professor** [comment]
I think the top level Makefile should have the targets that developers
routinely use. This is a common pattern on many software projects. It
documents what we expect developers to be able to do with the system.

**Pull request closed without merging**